src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
+			if (info) {
+				/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+				info?.values.forEach(([start, end]) => {
+					for (let i = start; i <= end; i++) {
+						// Adjust for 0-indexed array
+						values[i - 1] = true;
+					}
+				});
 
-			d.values.forEach(([start, end]) => {
-				for (let i = start; i <= end; i++) {
-					// Adjust for 0-indexed array
-					values[i - 1] = true;
+				return {
+					label: d,
+					values
+				};
+			} else {
+				if (d === 'fr') {
+					return allFractionData;
+				} else {
+					return {
+						label: d,
+						values: new Array(DATA_MAX).fill(true)
+					};
 				}
-			});
-
-			return {
-				label: d.label,
-				values
-			};
-		}),
-		fractionData
+			}
+		})
 	]);
+	let selection = $state({
+		start: 1,
+		end: 100
+	});
+	let detailData = $derived(
+		boolData.map((d) => {
+			return { label: d.label, values: d.values.slice(selection.start - 1, selection.end) };
+		})
+	);
 </script>
 
+<div class="card p-4 variant-filled-primary max-w-lg" data-popup="popupClick">
+	<p>
+		Um Textzeugen und Fragmente zu entfernen, klicken sie bitte einfach auf die grau hinterlegten
+		Kasten.
+	</p>
+	<p>
+		Um einzelne Fragmente hinzuzufügen geben sie <i>fr</i> gefolgt vom Index des Fragments ein. Z.B.
+		<i>fr32</i>.
+	</p>
+	<div class="arrow variant-filled-primary"></div>
+</div>
+<div
+	class="container mx-auto mb-6 flex flex-wrap md:flex-nowrap gap-4"
+	use:popup={{ event: 'focus-click', placement: 'top', target: 'popupClick' }}
```

Comment:
`placement: 'top'` seems to be ignored?!
Furthermore, I noticed that the popup can get hidden behind the topNavBar since the whole section is extending to the top of the viewport... consider adding a margin-top (maybe padding is enough?) to the main content div.

Reply:
It's not ignored. It is just a _preference_. If there is no space at the top, it will be placed elsewhere.
A simple margin is not possible (since the position of the popup is calculated...
I forced the position to top as a solution. see 5e968978b2313a4467ff3483ae547890ecc1a7fe

Reply:
> It's not ignored

Ah, now I understand the placement. So honestly, I'd leave it 

> A simple margin is not possible (since the position of the popup is calculated...

What I meant is adding a margin-top to the great-great-grandparent of the popup, such that it is not hiding behind the top bar. I think, if you then also adjust its **z-index** to the same value as the navbar, then the popup won't hide anymore.